examples/llava/clip.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CLIP_H
 #define CLIP_H
 
-#include "ggml.h"
+#include "llama.h"
```

Comment:
`clip.h` includes `llama.h` which is not great. The only reason is to be able to reuse the `LLAMA_API`.
Can be improved by having dedicated `CLIP_API`.

Reply:
Thanks for the review --I handled them and it should be ready as a shared lib now.